Content.Server/Medical/SuitSensors/SuitSensorSystem.cs

```diff
@@ -287,6 +300,35 @@ private string GetModeName(SuitSensorMode mode)
         return Loc.GetString(name);
     }
 
+    public void TrySetSensor(EntityUid uid, SuitSensorMode mode, EntityUid userUid,
+        SuitSensorComponent? component = null)
```

Comment:
Update TrySuitSensor and SetSensor's arguments to use `Entity<SuitSensorComponent?> entity` instead of having them split up as the first and fourth arguments

Reply:
If you can, just do `Entity<SuitSensorComponent>` without needing to resolve the component. I don't think SuitSensorComponent is ever null.